hydra/utils.py

```diff
@@ -12,38 +13,82 @@
 log = logging.getLogger(__name__)
 
 
+def _safeimport(path: str) -> Optional[ModuleType]:
+    """
+    Import a module; handle errors; return None if the module isn't found.
+    This is a typed simplified version of the `pydoc` function `safeimport`.
+    """
+    from importlib import import_module
+
+    try:
+        module = import_module(path)
+    except ImportError as e:
+        if e.name == path:
+            return None
+        else:
+            log.error(f"Error importing module: {path}")
+            raise e
+    except Exception as e:
+        log.error(f"Non-ImportError while importing module {path}: {e}")
+        raise ValueError(f"Non-ImportError while importing module {path}: {e}")
```

Comment:
the log.error is probably going to cause the message to be printed twice.

Reply:
You're right. I took it from the original file but I've noticed you've only used the log for BaseException errors that don't have a message attached.